contribs/gmf/apps/desktop/index.html

```diff
@@ -83,6 +83,14 @@
         </div>
       </div>
       <div class="map-container">
+        <div class="zoom-buttons" ngeo-recenter ngeo-recenter-map="::mainCtrl.map">
```

Comment:
Shouldn't we named it as map-buttons, because on mobile we as have the geolocation button in the div...